website/docs/community/join.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Join the community
+id: join
+---
+
+<section className="community-home">
+
+Want to learn how organizations around the world are tackling the biggest challenges in data while making new friends from the best analytics teams? Join the dbt community — data practitioners’ favorite place to learn new skills, grow your career, and forge connections.
+
+<div className="grid--3-col">
+
+<Card
+    title="Join us on Slack"
+    body="Follow the pulse of the dbt Community! Find the latest articles on analytics engineering, chat with other practitioners about your work or simply share a killer meme."
+    link="https://www.getdbt.com/community/"
+    icon="slack"
+/>
+
+<Card
+    title="Community forum"
+    body="Have a question about how to do something in dbt? Hop into our community forums and work with others to create long lived community knowledge."
+    link="forums"
+    icon="discussion"
+/>
+
+<Card
+    title="How to contribute"
+    body="Want to get involved? This is the place! Learn how to contribute to dbt open source. repositories, write for our blog, speak at a meetup and more."
+    link="community/contributing/contributing-intro" icon="writing"
```

Comment:
I think the intro is called "contribute" here: https://deploy-preview-2069--docs-getdbt-com.netlify.app/community/contribute
```suggestion
    link="community/contributing/contributing" icon="writing"
```